### DISCUSSION TITLE
What is the "last seen room temperature datetime"?

### DISCUSSION THREAD

**Discussion by Stooovie** [jmcollin78/versatile_thermostat#499] [Q&A]
I thought Better Thermostat was complex, but holy cow, I'm downright incapable of configuring this one. 

What is the "last seen room temperature datetime" supposed to be? I've searched, found nothing that would actually explain. 

Thanks!

**Comment by jmcollin78** [comment]
See https://github.com/jmcollin78/versatile_thermostat/issues/398
I will have to document this experimental feature. You can safely ignore it if you don't have it

**Comment by Stooovie** [comment]
Ah, thanks!
